src/main/scripts/includes/sdk_install_java

```diff
@@ -18,9 +28,7 @@ sdk_install_java() {
       curl -fSsL "https://get.sdkman.io" | bash
     fi
   fi
-  # This is a bit of a hack to avoid the interactive prompt but setting
-  # it on the commandline doesn't always work.
-  sed -e "s|sdkman_auto_answer=false|sdkman_auto_answer=true|g" -i ~/.sdkman/etc/config
+  __sdkman_batch_mode
```

Comment:
There's question as to whether this should get the current state, and then set it back to that, rather that assuming its already `=true`.

This could be a user could have there confirm reset for them later down the line.

Reply:
if it's already true, then it'll stay true with __sdk_batch_mode, but you're right, the sdk_interactive_mode should probably do a check to see if we turned it off, and only turn it back on if we toggled it.